File: src/ProjectR.Scripting/content/scripts/spells/SupportTechniqueWand.lua
function GetTargetType()
  return Allies
end

function GetName()
  return "Support Technique: Wand"
end

function GetDescription()
  return "Increases MD of all allies."
end

function IsSupportSpell()
    return false
end

function GetMPCost()
    return .8
end

function GetDelay()
  return .3
end

function GetSpellType()
  return Magical
end

function GetMasteries()
  return {  }
end

function SpellEffect()
  d:BuffStat(MD, .48)
end

File: src/ProjectR.Scripting/content/scripts/spells/SupportTechniqueWand.cs
﻿using System.Collections.Generic;
using ProjectR.Interfaces;
using ProjectR.Interfaces.Model;
using ProjectR.Interfaces.Model.Stats;

namespace ProjectR.Scripting
{
    public class SupportTechniqueWand : SpellScriptBase
    {
        public override string Name { get { return "Support Technique: Wand"; } }
        public override string Description { get { return "Increases MD of all allies."; } }

        public override TargetType TargetType { get { return TargetType.Allies; } }
        public override IList<EleMastery> Masteries { get { return new EleMastery[] { }; } }
        public override SpellType SpellType { get { return SpellType.Magical; } }
        public override bool IsSupportSpell { get { return false; } }

        public override double MPCost { get { return 80; } }
        public override double Delay { get { return .3; } }

        public override void SpellEffect(ICharacter caster, ICharacter target)
        {
            BuffStat(Stat.MD, .58);
        }
    }
}